sentinel_alert_rule/examples/simple/simple.tf

```diff
@@ -0,0 +1,7 @@
+
+module "simple" {
+  source = "../../"
+
+  billing_tag_value = "Terratest"
```

Comment:
This is no longer being used by anything, maybe we should delete the inputs, or re-enable the tags that it's associated with

Reply:
added a simple example